Tasks/PublishBuildArtifacts/task.json

```diff
@@ -12,11 +12,11 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 5
+    "Patch": 6
   },
   "demands": [
   ],
-    "minimumAgentVersion": "1.83.0",
+    "minimumAgentVersion": "1.91.0",
```

Comment:
We shouldn't need to bump the minimum agent version.  If it's the old agent, it will still run ps1.  If it's the new then this isn't needed.

Next sprint, we will remove the ps1 files for the node xplat type tasks and at that time (and only on those tasks) will be bump the minimum agent version